lib/simple_config.py

```diff
@@ -287,8 +287,17 @@ def static_fee_index(self, value):
     def has_fee_estimates(self):
         return len(self.fee_estimates)==4
 
-    def fee_per_kb(self):
-        return self.get('fee_per_kb', self.max_fee_rate()/2)
+    def custom_fee_rate(self):
+        f = self.get('customfee')
+        return f
+
+    def fee_per_kb(self): 
+       retval=self.get('customfee')
```

Comment:
Why is this not calling `custom_fee_rate` ?

Reply:
good catch

Reply:
Where is the config key 'customfee' getting set?

Will that be a future PR?

Reply:
One more issue:

1. Set the custom_fee_rate (fee slider disappears). Ok. √
2. Go to send tab.  Everything is peachy. Great! It even auto-updates the fee rate. √
3. Go back to prefs, clear the custom_fee_rate, fee_slider reappears and fee rate gets set to default (1000/kb). Great! √
4. Go back to send tab.. hover over fee slider.. it now reads 0.0 sats/byte (which is incorrect and is taking the older tooltip value of the hidden slider for some reason).